pkg/sql/crdb_internal.go

```diff
@@ -337,6 +338,57 @@ CREATE TABLE crdb_internal.tables (
 	},
 }
 
+var crdbInternalTablesTableLastStats = virtualSchemaTable{
+	comment: "the latest stats for all tables accessible by current user in current database (KV scan)",
+	schema: `
+CREATE TABLE crdb_internal.table_row_stats (
+  table_id         INT         NOT NULL,
+  table_name       STRING      NOT NULL,
+  row_count        INT         NOT NULL
+)`,
+	generator: func(ctx context.Context, p *planner, dbContext *sqlbase.ImmutableDatabaseDescriptor) (virtualTableGenerator, cleanupFunc, error) {
```

Comment:
you can just implement `populate` here, the API is a bit simpler.

Reply:
Fixed.